test test_save_glm_to_bids_glm_report_no_contrast failing

https://github.com/nilearn/nilearn/actions/runs/26817420307/job/79063010131#step:8:18725

```
  ________________ test_save_glm_to_bids_glm_report_no_contrast _________________
  [gw0] win32 -- Python 3.14.5 D:\a\nilearn\nilearn\.tox\plotting\Scripts\python.exe
  
  two_runs_model = FirstLevelModel(minimize_memory=False)
  tmp_path = WindowsPath('C:/Users/runneradmin/AppData/Local/Temp/pytest-of-runneradmin/pytest-1/popen-gw0/test_save_glm_to_bids_glm_repo0')
  
      @pytest.mark.slow
      @pytest.mark.thread_unsafe
      def test_save_glm_to_bids_glm_report_no_contrast(two_runs_model, tmp_path):
          """Run generate_report with no contrasts after save_glm_to_bids.
      
          generate_report tries to rely on some of the generated output,
          if no contrasts are requested to generate_report
          then it will rely on the content of
          model._reporting_data["filenames"]
      
          report generated by save_glm_to_bids should contain relative paths
          to the figures displayed as the report and its figures are meant
          to go together
      
          report generated after using save_glm_to_bids could be saved anywhere
          so evengthough we reuse pre-generated figures,
          we will rely on full path in this case
          """
          contrasts = {"BBB-AAA": "BBB-AAA"}
          contrast_types = {"BBB-AAA": "t"}
          model = save_glm_to_bids(
              model=two_runs_model,
              contrasts=contrasts,
              contrast_types=contrast_types,
              out_dir=tmp_path,
              **KWARGS,
          )
      
          assert model._reporting_data.get("filenames", None) is not None
      
          EXPECTED_FILENAMES = [
              "run-1_design.png",
              "run-1_corrdesign.png",
              "run-1_contrast-bbbMinusAaa_design.png",
          ]
      
          with (tmp_path / "report.html").open("r") as f:
  >           content = f.read()
                        ^^^^^^^^
  
  EXPECTED_FILENAMES = ['run-1_design.png', 'run-1_corrdesign.png', 'run-1_contrast-bbbMinusAaa_design.png']
  contrast_types = {'BBB-AAA': 't'}
  contrasts  = {'BBB-AAA': 'BBB-AAA'}
  f          = <_io.TextIOWrapper name='C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-runneradmin\\pytest-1\\popen-gw0\\test_save_glm_to_bids_glm_repo0\\report.html' mode='r' encoding='cp1252'>
  model      = FirstLevelModel(minimize_memory=False)
  tmp_path   = WindowsPath('C:/Users/runneradmin/AppData/Local/Temp/pytest-of-runneradmin/pytest-1/popen-gw0/test_save_glm_to_bids_glm_repo0')
  two_runs_model = FirstLevelModel(minimize_memory=False)
  
  nilearn\glm\tests\test_io.py:431: 
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
  
  self = <encodings.cp1252.IncrementalDecoder object at 0x000001E55BE5B3E0>
  input = b'<!DOCTYPE html>\n<html lang="en">\n    <head>\n        <title>Nilearn - Statistical Report - First Level Model</titl...n/nilearn/issues">here</a>.\n                </p>\n            </address>\n        </footer>\n    </body>\n\n\n</html>'
  final = True
  
      def decode(self, input, final=False):
  >       return codecs.charmap_decode(input,self.errors,decoding_table)[0]
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  E       UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 72789: character maps to <undefined>
  
  final      = True
  input      = b'<!DOCTYPE html>\n<html lang="en">\n    <head>\n        <title>Nilearn - Statistical Report - First Level Model</titl...n/nilearn/issues">here</a>.\n                </p>\n            </address>\n        </footer>\n    </body>\n\n\n</html>'
  self       = <encodings.cp1252.IncrementalDecoder object at 0x000001E55BE5B3E0>
  
  C:\hostedtoolcache\windows\Python\3.14.5\x64\Lib\encodings\cp1252.py:23: UnicodeDecodeError
  _ test_check_estimator_nilearn[estimator14-check_img_estimator_n_elements-check_img_estimator_n_elements] _
```

Seems to be an utf8 encoding error

Just not sure why this is showing up now though

--- a/nilearn/glm/tests/test_io.py
+++ b/nilearn/glm/tests/test_io.py
@@ -427,7 +427,7 @@ def test_save_glm_to_bids_glm_report_no_contrast(two_runs_model, tmp_path):
         "run-1_contrast-bbbMinusAaa_design.png",
     ]
 
-    with (tmp_path / "report.html").open("r") as f:
+    with (tmp_path / "report.html").open("r", encoding="utf-8") as f:
         content = f.read()
         assert "BBB-AAA" in content
         if is_matplotlib_installed():

